ts/lib/auth/googleauth.ts

```diff
@@ -411,7 +423,8 @@ export default class GoogleAuth {
    * @param {string} - The API key string
    * @param {function=} - Optional callback function
    */
-  public fromAPIKey(apiKey, opt_callback) {
+  public fromAPIKey(
+      apiKey: string, opt_callback: (err: Error, client?: JWTClient) => void) {
```

Comment:
I ensured the opt_callbacks were all actually optional, and renamed to just callback